packages/base/src/manager-base.ts

```diff
@@ -490,7 +490,7 @@ abstract class ManagerBase<T> {
      * @param data The initial data for the comm
      * @param metadata The metadata in the open message
      */
-    protected abstract _create_comm(comm_target_name: string, model_id: string, data?: any, metadata?: any): Promise<any>;
+    protected abstract _create_comm(comm_target_name: string, model_id?: string, data?: any, metadata?: any): Promise<any>;
```

Comment:
Note, the docs of `WidgetOptions` say that either `comm` or `model_id` should be supplied, but it is used without it e.g. here: https://github.com/jupyter-widgets/ipywidgets/blob/master/packages/controls/src/widget_controller.ts#L276  (and it still seems to work).